templates/state_bydate.js

```diff
@@ -0,0 +1,571 @@
+  {groupName:"Gas Stations",essential:true},
+  {groupName:"Government",essential:true},
+  {groupName:"Grocery",essential:true},
+  {groupName:"Hardware Stores",essential:true},
+  {groupName:"Hotel",essential:false},
+  {groupName:"Medical",essential:true},
+  {groupName:"Nightlife Spots",essential:false},
+  {groupName:"Office",essential:false},
+  {groupName:"Outdoors & Recreation",essential:false},
+  {groupName:"Professional & Other Places",essential:false},
+  {groupName:"Residences",essential:true},
+  {groupName:"School",essential:false},
+  {groupName:"Shops & Services",essential:false},
+  {groupName:"Spiritual Center",essential:false},
+  {groupName:"Sports",essential:false},
+  {groupName:"Travel & Transport",essential:false},
+  {groupName:"undefined",essential:false}
+  ];
+
+for (var groupIndex = 0; groupIndex < groupMappings.length; groupIndex++) {
+  var nextGroup = groupMappings[groupIndex];
+  groupToEssentialMap.set(nextGroup.groupName,nextGroup.essential);
+}
+
+var eventListener = function() {
+  county = countySel.value ? encodeURIComponent(countySel.value) : ALL
+  venue = locationTypeSel.value ? encodeURIComponent(locationTypeSel.value) : ALL;
+  windowLocationToSet = "/bydatesel/" + selectedState + "/" + county + "/" + venue;
+  if (urlParams.get('datafilename')) {
+    windowLocationToSet += "?datafilename=" + urlParams.get('datafilename');
+  }
+  window.location = windowLocationToSet;
+};
+
+function parseSelection() {
+  selectedState = (_selectedState == NONE || _selectedState =="") ? 'California' : _selectedState;
+  selectedCounties = _selectedCounties == ALL ? [] : _selectedCounties.split(",");
+  selectedVenues = _selectedVenues == ALL ? [] : _selectedVenues.split(",");
+}
+
+function setNavLinks() {
+  document.getElementById('nav-chartgrouped').href = "/bydatesel/" + encodeURIComponent(selectedState) + "/ALL/ALL";
+  document.getElementById('nav-chartall').href = "/bydatesel/" + encodeURIComponent(selectedState) + "/ALL/ALL?datafilename=raw";
+  document.getElementById('nav-stategrouped').href = "/bystatesel/" + encodeURIComponent(selectedState) + "/ALL";
+  document.getElementById('nav-stateall').href = "/bystatesel/" + encodeURIComponent(selectedState) + "/ALL?datafilename=raw";
+}
+
+essentialSel = document.getElementById('essential-select');
+essentialSel.addEventListener('change', function() {
+  redoFilter();
+  if (countySel.value || locationTypeSel.value) {
+    drawChart();
+  }
+});
+
+parseSelection();
+setNavLinks();
+if (!datafilename) {
+  datafilename = '{{foursquare_data_url}}/grouped' + selectedState.replace(/\s/g, '') + '.csv';
+    document.getElementById('nav-chartgrouped').classList.add('font-weight-bold')
+
+} else {
+  datafilename = '{{foursquare_data_url}}/' + datafilename + selectedState.replace(/\s/g, '') + '.csv';
+    document.getElementById('nav-chartall').classList.add('font-weight-bold')
+}
+Papa.parse(datafilename, {download: true, complete: parsingDone});
```

Comment:
What does this .js file have to do with adding faq to landing page nav? Seems like an accidental commit.

Reply:
You're right - this was an older file that was removed but accidentally left in local branch after synch. I removed it.